samples/beam/requirements.txt

```diff
@@ -0,0 +1,3 @@
+apache-beam==2.23.0
+google-cloud-bigtable==1.4.0
+google-cloud-core==1.3.0
```

Comment:
This should be pulled in as a transitive dependency via `google-cloud-bigtable`.